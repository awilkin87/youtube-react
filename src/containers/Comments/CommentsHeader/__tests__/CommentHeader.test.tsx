import { shallow } from "enzyme";
import CommentHeader from '../CommentsHeader';

describe('CommentHeader', () => {
  test('Renders CommentHeader with non-zero comment count', () => {
    const wrapper = shallow(<CommentHeader commentCount={1000} />);
    expect(wrapper).toMatchSnapshot();
  });

  
  test('Renders CommentHeader with zero comment count', () => {
    const wrapper = shallow(<CommentHeader commentCount={0} />);
    expect(wrapper).toMatchSnapshot();
  });
});
