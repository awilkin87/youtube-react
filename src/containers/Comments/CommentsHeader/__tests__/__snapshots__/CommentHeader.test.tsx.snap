// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentHeader Renders CommentHeader with non-zero comment count 1`] = `
<div
  className="container"
>
  <h4>
    1,000
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort By
  </Button>
</div>
`;

exports[`CommentHeader Renders CommentHeader with zero comment count 1`] = `
<div
  className="container"
>
  <h4>
    0
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort By
  </Button>
</div>
`;
